src/main/java/Eva.java

```diff
@@ -0,0 +1,73 @@
+import java.util.Scanner;
+
+public class Eva {
+
+    public static final String HORIZONTAL_LINE = "---------------------------------------------------------------";
```

Comment:
good job on initialising the horizontal line as a constant, with good naming conventions